src/js/controllers/preferencesFee.js

```diff
@@ -1,10 +1,14 @@
 'use strict';
 
-angular.module('copayApp.controllers').controller('preferencesFeeController', function($scope, $timeout, $ionicHistory, gettextCatalog, configService, feeService, ongoingProcess) {
+angular.module('copayApp.controllers').controller('preferencesFeeController', function($scope, $timeout, $ionicHistory, gettextCatalog, configService, feeService, ongoingProcess, popupService) {
 
   ongoingProcess.set('gettingFeeLevels', true);
-  feeService.getFeeLevels(function(levels) {
+  feeService.getFeeLevels(function(err, levels) {
```

Comment:
`getFeeLevels` function doesn't return `err`. Please, check the file `feeServices.js`